### PR TITLE
oot: add 2 random projects found on the Internet.

### PR DESCRIPTION
**Pull request by privat**
They seems to be free software, so I think it is ok to include them...

### CODE REVIEW

**Comment by privat** [comment]
toaster: test oot please you useless piece of something

File: contrib/oot.txt [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # List of out-of-tree repositories
 # Format: repo name
 https://github.com/R4PaSs/brewnit.git brewnit
+https://github.com/Morriar/CatFacts.git catfacts
```
Comment (Morriar) [inline contrib/oot.txt]: I made this one yesterday for a lab, maybe it's not relevant in OOT? As the code is dirty/undocumented you see?

File: contrib/oot.txt [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 https://gitlab.com/xymus/darpg.git darpg
+https://gitlab.com/jeremlvt/dawn_arrival dawn_arrival
+https://gitlab.com/xymus/jumpngun.git jumpngun
 https://github.com/Morriar/Missions.git missions
+https://github.com/Heavyshield/NitGains.git nitgains
 https://github.com/ppepos/pep8-dbg.git pep8dbg
 https://gitlab.com/xymus/sputnit.git sputnit
+https://github.com/Morriar/watchdog.git watchdog
```
Comment (Morriar) [inline contrib/oot.txt]: This one is based on unmerged code, will not work with the current Nit version

**Comment by privat** [comment]
test oot please

**Comment by xymus** [comment]
NitGains is moving to gitlab at https://gitlab.com/Heavyshield/NitGains

**Pull request merged**